src/models/validation-schema.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface ValidationSchema {
+    [key: string]: {
```

Comment:
**question (non-blocking):** is this formatted properly? I see indentation of 4 spaces instead of 2